staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -141,6 +141,13 @@ var (
 		},
 		[]string{"group", "version", "kind"},
 	)
+	WatchEvents = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "apiserver_watch_events_total",
+			Help: "Number of events in watches for a given resources",
```

Comment:
I bit more context in the help string might be useful, e..g "Number of events send to clients" ?

"for a given resources" might not be needed here? If a metric consumer filters by the labels they get per-resource, but if not they get the grand total.

Reply:
I corrected the help text. Please let me know if you find the new version more readable.